citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm

```diff
@@ -39,6 +39,24 @@
 		#set( $addH1SubsectionButtonId = 'addH1SubsectionButton' + $nestedSection.getLocation() )
 		#set( $buttonsDiv = 'buttonsDiv' + $nestedSection.getLocation() )
 		#set( $num = $num + 1)
+		#set( $citationNo = 0)
```

Comment:
Much better not todo this sort of code in a template, there's no go support for debugging. Much better to just have the Java calculate it and put the value in the context.